messages/german.json

```diff
@@ -0,0 +1,1393 @@
+        "crazed shaman": ["vs"],
+        "guardian angel": ["se"],
+        "mad scientist": ["vw"],
+        "matchmaker": ["vk"],
+        "vengeful ghost": ["rg"],
+        "village drunk": ["bs"]
+    },
+    "_commands": {
+        "*": "For assistance in configuring this section, please see https://werewolf.chat/Translation#Commands",
+        "": [""],
+        "abstain": ["verzichten", "vz", "keinlynch", "kl", "keinevote", "kv", "abstain", "abs", "nolynch", "nl", "novote", "nv"],
+        "admins": ["admins", "ops"],
+        "bite": ["beißen", "bite"],
+        "bless": ["segne", "bless"],
+        "cat": ["kat", "cat"],
+        "charm": ["bezaubern", "charm"],
+        "choose": ["wähle", "choose"],
+        "clone": ["kopiere", "clone"],
+        "coin": ["münze", "coin"],
+        "consecrate": ["weihe", "consecrate"],
+        "curse": ["verfluche", "curse"],
+        "deadchat": ["Todeschat", "deadchat"],
+        "eval": ["eval"],
+        "exec": ["exec"],
+        "faftergame": ["fnachspiel", "nachspiel", "faftergame", "aftergame"],
+        "fday": ["ftag", "fday"],
+        "fdie": ["fdie", "fbye"],
+        "fdo": ["fmessage", "fdo", "fme"],
+        "fflags": ["feinstellung", "fflags"],
+        "fgame": ["fspiel", "fgame"],
+        "fgoat": ["fziege", "fgoat"],
+        "fjoin": ["fbeitritt", "fjoin"],
+        "flastgame": ["fletztesspiel", "letztespiel", "flastgame", "lastgame"],
+        "fleave": ["fverlassen", "fleave", "fquit"],
+        "fnight": ["fnacht", "fnight"],
+        "force": ["force"],
+        "fpull": ["fbotupdate", "fpull", "pull"],
+        "freceive": ["fempfangen", "freceive"],
+        "frestart": ["fneustart", "frestart", "restart"],
+        "frole": ["frolle", "frole"],
+        "fsay": ["fsage", "fsay"],
+        "fsend": ["fsenden", "fsend"],
+        "fspectate": ["fspectate"],
+        "fstart": ["fstarten", "fstart"],
+        "fstasis": ["fspielsperre", "fstasis"],
+        "fstop": ["fstop"],
+        "ftemplate": ["ftemplate", "template"],
+        "ftotem": ["ftotem"],
+        "fwait": ["fwarten", "fwait"],
+        "fwarn": ["fwarn"],
+        "game": ["Spiel", "game"],
+        "games": ["Varianten", "Spiele", "games", "modes"],
+        "gamestats": ["Spielstats", "gamestats", "gstats"],
+        "goat": ["Ziege", "goat"],
+        "guard": ["schutz", "guard", "protect", "save"],
+        "help": ["hilfe", "help"],
+        "hex": ["verhex", "hex"],
+        "id": ["Suchen", "id", "investigate"],
+        "immunize": ["geben", "impfe", "give", "immunize", "immunise"],
+        "join": ["mm", "join", "j"],
+        "kill": ["Töte", "kill"],
+        "latency": ["Latenz", "latency"],
+        "leave": ["verlassen", "vl", "leave", "quit", "q"],
+        "lynch": ["galgen", "lynch"],
+        "match": ["verkupple", "match", "choose"],
+        "myrole": ["meinerolle", "myrole"],
+        "mystats": ["meinestats", "ms", "mystats", "m"],
+        "notice": ["notiz", "notice"],
```

Comment:
Shouldn't i? I think when I enter! Note the original notice command is still executed because Note is just an alias for notice or am I confusing something?